cli/src/service.rs

```diff
@@ -631,12 +615,7 @@ mod tests {
 			crate::chain_spec::kauri::config(),
 			|config| new_full(config),
 			|config| new_light(config),
-			vec![
-				"//Andrea".into(),
-				"//Brooke".into(),
-				"//Courtney".into(),
-				"//Drew".into(),
-			],
+			vec!["//Alice".into(), "//Bob".into(), "//Courtney".into(), "//Drew".into()],
```

Comment:
"Andrea", "Brooke"

Reply:
I have added these two to the list. Is that what you meant?

Reply:
I see Emily and Frank in some other places too. Should we add them as well?